Makefile

```diff
@@ -257,10 +257,15 @@ debian: sdist
 
 deb: deb-src
 	@for DIST in $(DEB_DIST) ; do \
+	    if test $${DIST} = trusty ; then \
+	        CRYPTO_BACKEND=pycrypto ; \
```

Comment:
Considering the dist is assumed to be an ubuntu release https://github.com/ansible/ansible/blob/devel/Makefile#L79 how would a numeric selection would be possible without overhauling the makefile?